html_css/css_basics.md

```diff
@@ -32,3 +32,4 @@ This section contains helpful links to other content. It isn't required, so cons
 
 * [Learn CSS Selectors Interactively Using CSS Diner](http://flukeout.github.io/)
 * [Smashing Magazine on Taming Advanced CSS Selectors](http://coding.smashingmagazine.com/2009/08/17/taming-advanced-css-selectors/)
+* [CSS Selectors Cheat Sheet](https://drive.google.com/file/d/1VXO0FyYk1vte0dVlDAYL03UK__RbFwTx/view?usp=sharing)
```

Comment:
I had closed your other PR https://github.com/TheOdinProject/curriculum/pull/22376 with this CSS cheat sheet. Please see the comments there as to why. A google drive link is also not ideal as whoever is hosting it could remove it at anytime along with this comment from that PR " 

"Update: I am closing this as learners can easily google the docs/cheat sheets (peer the teams consensus)."

If you remove this then I can approve the PR

Reply:
Done.